cinema/views.py

```diff
@@ -140,7 +164,7 @@
         )
     )
     serializer_class = MovieSessionSerializer
-    authentication_classes = (TokenAuthentication,)
+    # authentication_classes = (TokenAuthentication,)
```

Comment:
The `authentication_classes` line is commented out. If token authentication is necessary for the `MovieSessionViewSet`, consider uncommenting this line.